tendrl/ceph_integration/objects/rbd/atoms/create/__init__.py

```diff
@@ -13,6 +17,54 @@ def __init__(self, *args, **kwargs):
         super(Create, self).__init__(*args, **kwargs)
 
     def run(self):
+        # Checking if a new  pool has to be created or not rbd creation.
+        if self.parameters['Rbd.create_pool'] is True and \
+                        self.parameters['Rbd.pool_id'] is None:
+            # Creating job for pool creation.
+            if ['Rbd.pool_parameters'] in self.parameters:
```

Comment:
The reason I had mentioned use of pool_parameters as a dict which contains the pool parameters is the pool creation can have more optional parameters other than the 4 mentioned above. https://github.com/Tendrl/ceph-integration/blob/develop/tendrl/ceph_integration/objects/definition/ceph.yaml#L286